backend/dataline/services/llm_flow/toolkit.py

```diff
@@ -487,6 +489,7 @@ def get_response(  # type: ignore[misc]
         res = generate_chart_call.extract()
 
         # Find the last data result
+        # TODO: WHY IS THIS NOT TRIGGERING?
```

Comment:
ummm what's supposed to trigger?

Reply:
ah nothing a specific bug im investigating, will remove in later PR